test/conformance/helpers/broker_tracing_test_helper.go

```diff
@@ -22,6 +22,7 @@ import (
 	"testing"
 
 	ce "github.com/cloudevents/sdk-go"
```

Comment:
To remove this we need to port the event transformation pod right?

Reply:
yep.